package.json

```diff
@@ -108,9 +108,40 @@
     "version-check": "node ./scripts/tasks/version-check.js"
   },
   "jest": {
-    "projects": [
```

Comment:
Yup, it's a known issue you already filed https://github.com/facebook/jest/issues/4117 😉